.github/workflows/vagrant-up.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: vagrant-up
+on:
+  - push
+  - pull_request
+
+jobs:
+  vagrant-up:
+    runs-on: macos-10.15
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: "${{ github.repository }}"
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Cache Vagrant boxes
+        uses: actions/cache@v2
+        with:
+          path: ~/.vagrant.d/boxes
+          key: ${{ runner.os }}-vagrant-${{ hashFiles('Vagrantfile') }}
+          restore-keys: |
+            ${{ runner.os }}-vagrant-
+
+      - name: Run vagrant init
```

Comment:
@lincmba do we need the `Vagrant init` step?  We already have a Vagrant file, no?